src/engine/websites/decorators/MadTheme.ts

```diff
@@ -0,0 +1,80 @@
+import { FetchRequest, FetchCSS } from '../../FetchProvider';
+import { type MangaScraper, type DecoratableMangaScraper, type Manga, Chapter, type Page } from '../../providers/MangaPlugin';
+import * as Common from './Common';
+
+const scriptImageLinks = `
+    new Promise(resolve => {
+        const images = window.chapImages.split(',');
+        resolve(images.map(image => window.mainServer + image));
+    });
+`;
+
+/* eslint-disable-next-line @typescript-eslint/no-explicit-any */ //=> A mixin class must have a constructor with a single rest parameter of type 'any[]'
+export type Constructor = new (...args: any[]) => DecoratableMangaScraper;

```

Comment:
Is the export necessary, i didn't see any imports for this type?
Is this type definition necessary, can the type `Common.Constructor` be used instead?

Reply:
Ill fix that